scripts/pylint_extensions.py

```diff
@@ -1255,7 +1255,7 @@ def process_module(self, node):
                     'no-space-at-beginning', line=line_num + 1)
 
             # Check if comment contains version info or data type.
-            if not previous_line.startswith(b'#'):
+            if line.startswith(b'#'):
```

Comment:
Sorry for the late reply -- I was travelling last weekend.

Can you explain in more detail the logic here? I understand the previous checks but I don't quite understand the logic behind this one. The comment before this line should explain why it is OK to skip further checks for this line.

Reply:
It checks if comment contains any data structure(str, int, etc.) at the beginning of comment then skip the check for capital letter.